lib/core-events/src/index.ts

```diff
@@ -16,6 +16,8 @@ enum events {
   FORCE_RE_RENDER = 'forceReRender',
   // Force the current story to re-render from scratch, with its initial args
   FORCE_REMOUNT = 'forceRemount',
+  // Request the story has been loaded into the store, ahead of time, before it's actually
```

Comment:
The sentence ends abruptly :) *before it's actually* what?

Also, that is a missing word between `story` and `has`, like `which`